src/utils/formContext.ts

```diff
@@ -143,9 +143,9 @@ export const getFormContext = (formUuid: string) => {
     ) => formState.runIntegrationActions(actionIds, options),
     runAIExtraction: async (
       extractionId: string,
-      waitForCompletion = false,
+      options = { waitForCompletion: true },
       pages?: number[]
-    ) => formState.runAIExtraction(extractionId, !waitForCompletion, pages),
+    ) => formState.runAIExtraction(extractionId, options, pages),
```

Comment:
doesn't waitForCompletion still need to default to false to be backwards compatible?